frontend/packages/console-dynamic-plugin-sdk/scripts/create-pkg.ts

```diff
@@ -0,0 +1,26 @@
+import * as fs from 'fs';
+import * as readPkg from 'read-pkg';
+
+const PKG_NAME = '@openshift-console/dynamic-plugin-sdk';
+const EXTENSIONS_EXPORT = 'lib/extensions/index.js';
+const WEBPACK_PLUGIN_EXPORTS = './lib/webpack/ConsoleRemotePlugin.js';
+
+const createPackageJson = () => {
```

Comment:
We should look into maintaining a static package.json with proper values and up to date version.
This could be done by separating out everything that shouldn't be in the SDK into a separate `plugin-runtime` package. The idea being that everything in this package is only for consumption as public API.

Reply:
OK, that will simplify things. I will look into it in a followup